e2e/main_test.go

```diff
@@ -36,8 +44,44 @@ func TestShimLoggers(t *testing.T) {
 		if *LogDriver == awslogsDriverName || *LogDriver == "" {
 			testAwslogs()
 		}
+		if *LogDriver == fluentdDriverName || *LogDriver == "" {
+			testFluentd()
+		}
 	})
 
 	gomega.RegisterFailHandler(ginkgo.Fail)
 	ginkgo.RunSpecs(t, description)
 }
+
+func sendTestLogByContainerd(creator cio.Creator, testLog string) {
+	// Create a new client connected to the containerd daemon
+	client, err := containerd.New(containerdAddress)
+	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+	defer client.Close()
+	// Create a new context with a customized namespace
+	ctx := namespaces.WithNamespace(context.Background(), "testShimLoggers")
+	// Pull an image
+	image, err := client.Pull(ctx, testImage, containerd.WithPullUnpack)
+	gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+	// Create a new container with the pulled image
+	container, err := client.NewContainer(ctx, testContainerId+"9", containerd.WithImage(image),
+		containerd.WithNewSnapshot("test-snapshot-9", image), containerd.WithNewSpec(oci.WithImageConfig(image),
```

Comment:
what's the purpose of the "9" appended to the end of the container ID and snapshot?

Reply:
Oh leftover for debugging.. thanks for pointing it out.